Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1

```diff
@@ -462,3 +462,69 @@ function Restore-VmfsVolume {
 
     $Esxi | Get-VMHostStorage -RescanVMFS -ErrorAction stop | Out-Null
 }
+
+<#
+    .DESCRIPTION
+     Renames an existing VMFS Datastore
+
+    .PARAMETER ClusterName
+     Cluster name
+
+    .PARAMETER DeviceNaaId
+     NAA ID of device associated with the existing VMFS volume
+
+     .PARAMETER DatastoreName
+     New datastore name.
+
+    .EXAMPLE
+     Set-VmfsDatastoreName -ClusterName "myClusterName" -DeviceNaaId $DeviceNaaId -DatastoreName $NewDatastoreName
+
+    .INPUTS
+     vCenter cluster name and device NAA ID.
```

Comment:
Could we also add datastore name as an input here?

Reply:
Sure